database/migrations/2025_12_10_000000_add_count_last_news_post_published_at_notified.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+declare(strict_types=1);
+
+use App\Models\Count;
+use App\Models\NewsPost;
+use Illuminate\Database\Migrations\Migration;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Count::firstOrCreate(
+            ['name' => 'last_news_post_published_at_notified'],
+            ['count' => NewsPost::default()->first()?->published_at->timestamp ?? 0],
```

Comment:
I mean in the `Count` model function itself instead of creating a dedicated migration 🤨 

seems a bit silly adding more migration just for this